packages/flutter/lib/src/widgets/async.dart

```diff
@@ -383,52 +406,25 @@ class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
 ///
 /// ## Sample code
 ///
-/// This sample shows a button used to initiate an asynchronous computation
-/// with a string result, and a text label reflecting the progress:
+/// This sample shows a [FutureBuilder] configuring a text label to show the
+/// state of an asynchronous calculation returning a string. Presumably,
+/// the `_calculation` field is set by pressing a button elsewhere in the UI:
```

Comment:
this is a total nit and feel free to ignore it if you disagree: I think I would say "Assume" instead of "Presumably,", and I would replace the colon with a period.